documentation_templates/tools.md

```diff
@@ -3,156 +3,109 @@
 
----
 
-## Quick start guide
+## How to cite this software
 
-> General guide for deployment across multiple infrastructures (distinct from specific infrastructure quick start guide).
 
----
+## User guide
 
-## Infrastructure usage and recommendations
 
-> You could include:
-> + a link to installation instructions for each computational infrastructure 
-> + recommendations for use on a specific computational infrastructure
+### Install
 
-> Documentation for a specific infrastructure should go into a infrastructure documentation template https://github.com/AustralianBioCommons/doc_guidelines/blob/master/infrastructure_optimisation.md
-
----
+> General installation guide.
 
-## Compute resource usage across tested infrastructures
+> If there are different installation requirements based on infrastructures you could indicate these here, or in the individual infrastructure documentation template: https://github.com/AustralianBioCommons/doc_guidelines/blob/master/infrastructure_optimisation.md
 
-> Table with high level compute resource usage information for standalone runs or testing of specific versions on specific computational infrastructures.
 
-| Tool | Version | Sample description | Wall time | Cores | Peak RAM in GB (requested) | Drive (GB) | HPC-HTC | If HPC-HTC is other, specify | Scheduler | Year-Month |
-| ----- | ------- | ------------------ | --------- | ----- | -------------------------- | ---------- | ------- | ---------------------------- | --------- | ---------- |
-|       |         |                    |           |       |                            |            |         |                              |           |            |
+### Quick start guide
 
----
+> General guide for deployment across multiple infrastructures (distinct from specific infrastructure quick start guide).
 
-## Benchmarking
 
-> Benchmarking for a specific infrastructure should go here: if this document is complicated it should go into a benchmarking template, or be provided elsewhere (e.g. Zenodo).
+### Required (minimum) inputs / parameters
 
----
+> The minimum inputs required for the workflow to run.
 
-# Tool summary
 
-## Metadata
+### Third party tools / dependencies
 
-> Example table below - **based on requirements for [bio.tools](https://bio.tools/)**
+> Add / list known dependencies, or link to a list of these dependencies.
 
-> **To discover appropriate [EDAM](https://github.com/edamontology/edamontology) ontology terms**, you can use [EDAM browser](https://edamontology.github.io/edam-browser/) 
 
-> bio.tools citation: Ison, J. *et al.* (2015). [Tools and data services registry: a community effort to document bioinformatics resources.](http://nar.oxfordjournals.org/content/early/2015/11/03/nar.gkv1116.long) _Nucleic Acids Research_.
- doi: [10.1093/nar/gkv1116](http://dx.doi.org/10.1093/nar/gkv1116) PMID: [26538599 ](http://www.ncbi.nlm.nih.gov/pubmed/26538599)
+### Infrastructure usage and recommendations
```

Comment:
For tools, I think its more informative to focus on operating system than infrastructure in most instances.

Reply:
Unless we're talking about mentioning the availability of the tool on a particular infrastructure, say something thats installed in if89 or globally on Gadi.